pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -132,25 +132,22 @@ public String toExplainString() {
   @Override
   protected TransferableBlock getNextBlock() {
     try {
```

Comment:
Do we always need to do this try-catch? Consider moving it into base class?

Reply:
yes i can move it to base